reference/safe-core-sdk/onramp-kit/stripe.md

```diff
@@ -0,0 +1,93 @@
+## Stripe
+
+The [Stripe Crypto OnRamp service](https://stripe.com/docs/crypto/overview) allows individuals to purchase cryptocurrencies securely from your platform or Dapp. It is fully customizable and can be easily integrated into your product or service.
+
+### Install
+
+If you choose to use the `StripePack`, you must install additional dependencies in addition to the `@safe-global/onramp-kit` package.
+
+Install the `@stripe/stripe-js` and `@stripe/crypto` packages:
```

Comment:
See other comments. This is self-explanatory and self-evident from the title and command that follows.

```suggestion
```